### PR TITLE
hyper_L documentation fix

### PR DESCRIPTION
**Pull request by LiMaIG**

### CODE REVIEW

File: include/deal.II/grid/grid_generator.h [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // the top level directory of deal.II.
 //
 // ---------------------------------------------------------------------
-
+
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: Would you please remove this extra space?

File: doc/news/changes/minor/20190807ImbertGerard.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+changed the comment to the hyper_L function
+ 
+(Lise-Marie Imbert-Gerard, August 7, 2019)
```
Comment (tjhei) [inline doc/news/changes/minor/20190807ImbertGerard.txt]: Please use the same format as the other files in the same folder (date format and use ``<br>``, ...)

**Comment by drwells** [comment]
Looks good. Would you please rebase your commits together?

**Comment by bangerth** [comment]
@LiMaIG -- nice first pull request! Talk to me tomorrow and we'll address the changes requested by the others!

**Comment by LiMaIG** [comment]
@bangerth is sitting next to me and says this is ok to merge!

**Comment by bangerth** [comment]
@tjhei -- please release your lock on this PR and merge yourself!

**Comment by drwells** [comment]
Ping @tjhei :)

**Comment by Rombur** [comment]
This PR breaks the indent script for me. Some how LMIG and LiMaIG end up in the list of names that we check in `contrib/utilities/indent_common.sh` and hell breaks loose. I get `invalid author 'LMIG' without firstname and lastname` when trying to apply the indentation. I haven't followed what you did for the indent script so I am not sure how to fix this.

**Comment by tjhei** [comment]
@LiMaIG can you please check that your git username is set to your full "first name last name"? https://help.github.com/en/articles/setting-your-username-in-git

**Comment by LiMaIG** [comment]
@tjhei I just updated my username

**Pull request merged**